packages/web-components/src/badge/badge.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { expect, test } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { fixtureURL } from '../helpers.tests.js';
+import type { Badge } from './badge.js';
+
+test.describe('Badge component', () => {
+  let page: Page;
+  let element: Locator;
+  let root: Locator;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+
+    element = page.locator('fluent-badge');
+    root = page.locator('#root');
+
+    await page.goto(fixtureURL('components-badge--badge'));
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
```

Comment:
We might consider adding a test for the default content to make sure it renders properly in the slot.